_sass/oscar.scss

```diff
@@ -137,19 +137,6 @@ details.spoiler [open] summary::after {
   font-size:2em;
 }
 
-.failure{
-  display: none;
-}
-
-
-div.borked > div.gallery_example_cards > div.item_example_card.failure{
-  display: block;
-}
-
-div.borked > div.gallery_example_cards > div.item_example_card{
-  display: none;
-}
-
```

Comment:
If you change `oscar.scss` I recommend you also change `_includes/head.html` to increment the "version", i.e., in this line:
```
  <link rel="stylesheet" href="{{ site.baseurl }}/public/css/oscar.css?version=3">
```
this ensures that everyone gets to "see" the new CSS without all kinds of reloads and other shenanigans as I had ot perform on PR #617